src/Tribe/Editor/Template/Overwrite.php

```diff
@@ -191,34 +194,40 @@ public function has_tickets_support( $post_id ) {
 	 * @param $post_id
 	 */
 	public function setup_template( $post_id ) {
-		tribe( 'tickets.editor.template' )->add_template_globals( array(
-			'post_id' => $post_id,
-			'post'    => get_post( $post_id ),
-		) );
+		/** @var Tribe__Tickets__Editor__Template $template */
+		$template = tribe( 'tickets.editor.template' );
+
+		$template->add_template_globals(
+			[
+				'post_id' => $post_id,
+				'post'    => get_post( $post_id ),
+			]
+		);
 	}
 
 	/**
-	 * Remove the actions and filters used to attach the classic editor views associated with
-	 * tickets
+	 * Remove the actions and filters used to attach the classic editor views associated with tickets.
 	 *
 	 * @since 4.9
 	 */
 	public function remove_classic_views() {
-		if ( ! ( $this->ticket_type instanceof Tribe__Tickets__Tickets ) ) {
+		if ( ! $this->ticket_type instanceof Tribe__Tickets__Tickets ) {
 			return;
 		}
 
 		$ticket_form_hook = $this->ticket_type->get_ticket_form_hook();
 		if ( ! empty( $ticket_form_hook ) ) {
-			remove_action( $ticket_form_hook, array(
+			remove_action(
+				$ticket_form_hook, [
```

Comment:
:no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: d3f7d5c0b309dab2a89228ba7917b583</sub>